meshtastic/tests/test_examples.py

```
@@ -52,3 +63,98 @@ def test_examples_hello_world_serial_with_arg(
+    assert isinstance(device_cfg.get("serial_enabled"), bool)
+    assert "serialEnabled" not in device_cfg
+
+    display_cfg = _require_mapping(
+        config_section.get("display"), label="config.display"
+    )
+    assert isinstance(display_cfg.get("screen_on_secs"), int)
+    assert "screenOnSecs" not in display_cfg
+
+    lora_cfg = _require_mapping(config_section.get("lora"), label="config.lora")
+    region = lora_cfg.get("region")
+    assert isinstance(region, str) and region
```

Ruff (0.15.4)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)